app/pages/api/organizations/index.js

```diff
@@ -0,0 +1,33 @@
+import dbConnect from '../../../middleware/dbConnect';
+import { getNewOrgId } from '../../../middleware/helpers';
+import Org from '../../../models/Org';
+
+/**
+ * Returns one 'organization' document, may be null
+ *
+ */
+export default async function handler(req, res) {
+  const { method } = req;
+  await dbConnect();
+
+  const id = await getNewOrgId(req.body?.name);
+
+  switch (method) {
+    case 'POST':
+      try {
+        const org = await Org.create({
+          ...req.body,
+          id,
+        });
+
+        res.status(201).json({ success: true, data: org });
+      } catch (error) {
+        res.status(400).json({ success: false });
```

Comment:
Can we include the error in the response as such: `.json({ success: false, message: error.message ?? "" })`